sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/DAGWriterJsonTest/clickstream-teaser-clickstream-teaser-package.txt

```diff
@@ -28,7 +28,7 @@
   "stage" : "flink",
   "inputs" : [ "click_1" ],
   "plan" : "LogicalTableScan(table=[[click_1]], hints=[[[WatermarkHint inheritPath:[] options:[1]]]]) hints[WatermarkHint options:[1]]\n",
-  "sql" : "SELECT *\nFROM `click_1`",
+  "sql" : "SELECT *\nFROM `click_1`\n/*+ `WatermarkHint` */",
```

Comment:
I shouldn't be printing the hint here. I wonder if something changed in the SQL printer for Calcite.